agentarchives/atom/client.py

```diff
@@ -42,9 +44,10 @@ class AtomClient(object):
     This change is due to the fact that slugs are visible by users whereas IDs aren't.
     """
 
-    def __init__(self, url, key):
+    def __init__(self, url, key, timeout=None):
         self.key = key
         self.base_url = urljoin(url, 'api/')
+        self.timeout = timeout or DEFAULT_TIMEOUT
```

Comment:
You're right. Actually I think I'm going to change it so the func parameter defaults to `DEFAULT_TIMEOUT`. I think this is a sane default given what's said in the docs:

> `timeout` is not a time limit on the entire response download; rather, an exception is raised if the server has not issued a response for `timeout` seconds (more precisely, if no bytes have been received on the underlying socket for `timeout` seconds).

What do you think?